src/components/core/DpDataTable/DpDataTable.vue

```diff
@@ -36,68 +36,21 @@
       </thead>
 
       <!-- not draggable -->
-      <tbody v-if="!isDraggable">
-        <template
-          v-if="!isLoading && items.length > 0"
-          v-for="(item, idx) in items">
-          <dp-table-row
-            :index="idx"
-            :checked="elementSelections[item[trackBy]] || false"
-            :fields="fields"
-            :has-flyout="hasFlyout"
-            :header-fields="headerFields"
-            :is-draggable="isDraggable"
-            :is-expandable="isExpandable"
-            :is-loading="isLoading && items.length > 0"
-            :is-locked="lockCheckboxBy ? item[lockCheckboxBy] : false"
-            :is-locked-message="mergedTranslations.lockedForSelection"
-            :is-resizable="isResizable"
-            :is-selectable="isSelectable"
-            :is-selectable-name="isSelectableName"
-            :is-truncatable="isTruncatable"
-            :item="item"
-            :search-term="searchTerm"
-            :track-by="trackBy"
-            :wrapped="wrappedElements[item[trackBy]] || false"
-            @toggle-expand="toggleExpand"
-            @toggle-select="toggleSelect"
-            @toggle-wrap="toggleWrap">
-            <template v-slot:[field]="item" v-for="field in fields">
-              <slot
-                :name="field"
-                v-bind="item" />
-            </template>
-            <template v-slot:flyout="item">
-              <slot
-                name="flyout"
-                v-bind="item" />
-            </template>
-          </dp-table-row>
-
-          <!-- DpTableRowExpanded -->
```

Comment:
Thanks a lot for your support! I noticed a problem when using the` <component> ` to define the tag (either `dp-draggable` or `tbody`). The `dp-draggable `component does not work correctly with the expanded functionality when the expanded row is inside the `DpDraggable` component.

There is no `<!-- DpTableRowExpanded --> `part in your code. If I add expanded row to the `<component>` below the `<dp-table-row>`, the draggable functionality does not work even when I add `v-if="!isDraggable"` to the expanded row.

Reply:
The previous version worked well because the expanded row was only in the `tbody` section.